src/chrome/content/rules/LeaseWeb.xml

```diff
@@ -9,23 +9,27 @@
 		- secure.leaseweb.com
 
 -->
-<ruleset name="LeaseWeb.com (partial)">
+<ruleset name="LeaseWeb (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="leaseweb.com" />
-	<target host="secure.leaseweb.com" />
 	<target host="www.leaseweb.com" />
+	<target host="auth.leaseweb.com" />
+	<target host="mirror.leaseweb.com" />
+	<target host="secure.leaseweb.com" />
+	
+	<target host="*.leaseweb.net" />
```

Comment:
This wildcard breaks various hosts.

Reply:
How so ?

Reply:
Here are some:
```
ns7.leaseweb.net
ns8.leaseweb.net
www.leaseweb.net (same for ^ btw.)
www.mirror.de.leaseweb.net

Reply:
Can you add a comment for them on the top?